Gauntlet.toml

```diff
@@ -2436,235 +2436,240 @@ document = "broadinstitute/warp:/verification/test-wdls/TestWholeGenomeGermlineS
 message = "TestWholeGenomeGermlineSingleSample.wdl:54:45: error: type mismatch: expected type `DragmapReference`, but found type `DragmapReference?`"
 permalink = "https://github.com/broadinstitute/warp/blob/30e4994f2585ff5ea429e6435acb2312810ef426/verification/test-wdls/TestWholeGenomeGermlineSingleSample.wdl/#L54"
 
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/amr/run.wdl"
+message = "run.wdl:146:24: error: type mismatch: expected type `Array[File]+`, but found type `Array[File]`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/a04293a5271176885ce7f876b6353b20da3f7b98/workflows/amr/run.wdl/#L146"
```

Comment:
Likewise, this error was hidden by a previous return of `Union`, but now `select_all` is returning `Array[File]` despite it being invalid.